azkaban-common/src/main/java/azkaban/constants/ServerInternals.java

```diff
@@ -26,5 +26,9 @@
   public static final String AZKABAN_EXECUTOR_PORT_FILENAME = "executor.port";
 
   public static final String AZKABAN_SERVLET_CONTEXT_KEY = "azkaban_app";
-  
+
+  //property names
+  public static final String AZKABAN_HOST_ALIAS = "azkaban.hostalias";
```

Comment:
This config is exposed to the users. Shouldn't it be in azkaban.constants.ServerProperties and follow the naming convention there?